### ISSUE TITLE
"Invalid array length" bug during response transpose

### ISSUE THREAD

**Issue by knownasilya** [AppGeo/maplarge#11]
This code returns `Infinity`, but it should never https://github.com/AppGeo/maplarge/blob/master/lib/index.js#L364

![screen shot 2018-12-21 at 3 16 02 pm](https://user-images.githubusercontent.com/34726/50361913-7ec7b900-0533-11e9-8881-608627422ea3.png)
![screen shot 2018-12-21 at 3 16 10 pm](https://user-images.githubusercontent.com/34726/50361914-7ec7b900-0533-11e9-86c7-66d8358c6d4b.png)

**Comment by calvinmetcalf** [comment]
yeah so looks like it doesn't take into account that the array might be empty

**Comment by knownasilya** [comment]
Maybe default value for that reduce should be `0`?
